.github/workflows/publish-pr-test-results.yml

```diff
@@ -0,0 +1,30 @@
+name: Publish Pull Request Test Results
+
+on:
+  workflow_run:
+    workflows: ["Receive Pull Request"]
+    types:
+      - completed
+
+jobs:
+  publish-test-results:
+    runs-on: ubuntu-latest
+    # the build-and-test job might be skipped, we don't need to run this job then
+    if: >
```

Comment:
is the precendence here `PR && (success || failure)` ? 
If so, are there any other conclusions (other than `success` or `failure`)?

Reply:
I think there are [more](https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#job-status-check-functions) but we don't want to publish the test results if for example status of the build-and-test workflow was cancelled.

Reply:
@AMCN41R I have gone a head and added the logical seperators, to make the precedence clearer.

Reply:
yeah thats fine - only reason I asked was that if they are the only 2 conclusions, the condition isn't required - looks good